CHANGELOG.md

```diff
@@ -1,11 +1,22 @@
-## 2.7.0 beta v1 (March 2018)
+## 2.7.0 (Abril 2018)
```

Comment:
Set  the month name in english (April), please.

Reply:
Sure 💃